packages/input-select/src/components/withImage.js

```diff
@@ -1,10 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { badgeSizes } from "@crave/farmblocks-image";
+import Image, { badgeSizes } from "@crave/farmblocks-image";
+import styled from "styled-components";
 
-import ItemImage from "../styledComponents/ItemImage";
 import LabelContainer from "../styledComponents/LabelContainer";
 
+const ItemImage = styled(Image)`
+  margin-right: -7px;
+  margin-left: 10px;
```

Comment:
Field is being rendered over the image (see story: `withImage - Valid image path`):
![captura de tela de 2018-07-18 09-57-06](https://user-images.githubusercontent.com/17071199/42883096-f83d8db6-8a70-11e8-9edf-89544a552e44.png)


Master branch:
![captura de tela de 2018-07-18 09-56-35](https://user-images.githubusercontent.com/17071199/42883072-e90ab878-8a70-11e8-859d-512ca2508ee3.png)

Reply:
`SelectInput` is designed to work with a `formInput` field. This story was written with a regular input, so I think we are good here as long as the select input with images looks right.

![image](https://user-images.githubusercontent.com/17936244/42883704-b582336c-8a72-11e8-9fc3-5902f5d3d590.png)